packages/react/src/patterns/blocks/LeadSpaceBlock/LeadSpaceBlock.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import {VideoPlayer} from "../../../components/VideoPlayer";
+import { LinkList } from '../../sub-patterns/LinkList';
+import { Button} from 'carbon-components-react';
```

Comment:
Actually now that I'm looking closer, shouldn't you be using the `CTA` in the `ContentBlock` instead?

Reply:
@jeffchew yeah, I thought of using CTA but it has the arrow icon with it, and in the sketch its mentioned as button so I have used primary button here. Can I use CTA just for text?

Reply:
Adding @annawen1 here, I think we might need to add a button type of "none" to not include any icon here perhaps? I think this was mentioned for another pattern as well if I'm not mistaken?

Reply:
yeah it was mentioned in the QA calls, we will need to add that option to the CTA component for ButtonGroup - should edit to use CTA for now, and then open a ticket to allow for no icon?

Reply:
@jeffchew shall I use CTA there or just leave it as button?

Reply:
@greeshmareddyg just use CTA for now. We will create a separate PR to update CTA

Reply:
@jeffchew used CTA.

Reply:
It looks like `Button` is still getting imported?

Reply:
removed all the button data in the file and documentation.